lib/core/MongoDbConfirmationStore.ts

```diff
@@ -0,0 +1,58 @@
+import IConfirmationStore from './interfaces/IConfirmationStore';
+import MongoDbStore from '../common/MongoDbStore';
+
+interface ConfirmationModel {
+  anchorString: string;
+  submittedAt: number;
+  confirmedAt: number | null;
+}
+
+/**
+ * Implementation of LastWriteStore that stores the last update per write
+ */
+export default class MongoDbConfirmationStore extends MongoDbStore implements IConfirmationStore {
+  public static readonly collectionName: string = 'confirmations';
+
+  constructor (serverUrl: string, databaseName: string) {
+    super(serverUrl, MongoDbConfirmationStore.collectionName, databaseName);
+  }
+
+  public async confirm (anchorString: string, confirmedAt: number): Promise<void> {
+    await this.collection.updateMany({ anchorString }, { $set: { confirmedAt } });
+  }
+
+  public async resetAfter (confirmedAt: number | undefined): Promise<void> {
+    await this.collection.updateMany({ confirmedAt: { $gt: confirmedAt } }, { $set: { confirmedAt: null } });
```

Comment:
Add a unit test for `undefined`, I think it's edge case, but since it can be passed, should clear the entire collection in which case.